x-pack/test/functional/apps/ml/data_visualizer/index_data_visualizer.ts

```diff
@@ -220,5 +221,55 @@ export default function ({ getPageObject, getService }: FtrProviderContext) {
       });
       runTests(sampleLogTestData);
     });
+
+    describe('with view in lens action ', function () {
+      const testData = farequoteDataViewTestData;
+      // Run tests on full ft_module_sample_logs index.
+      it(`${testData.suiteTitle} loads the data visualizer selector page`, async () => {
+        // Start navigation from the base of the ML app.
+        await ml.navigation.navigateToMl();
+        await ml.navigation.navigateToDataVisualizer();
+      });
+
+      it(`${testData.suiteTitle} loads the source data in the data visualizer`, async () => {
```

Comment:
This test actually checks that lens charts are loading, so could be something like
```suggestion
      it(`${testData.suiteTitle} loads lens charts`, async () => {
```